js/src/ui/SelectionList/selectionList.css

```diff
@@ -58,7 +79,3 @@
   filter: grayscale(10%);
   opacity: 0.75;
```

Comment:
There are disabled accounts. Hardware wallets show up as disabled when they are not plugged in. (Same on the signer - when not in it is disabled.)

My preference for the borders (and actual icon colours) would be the "almost yellow" as in the CreateAccount dialog icons. But that needs some playing. (Basically the opposite of the link blue)